src/lightning_habana/__init__.py

```diff
@@ -1,31 +1,16 @@
-"""Root package info."""
+# Copyright (c) 2023 Habana Labs, Ltd. an Intel Company
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
-import os
-
-_PACKAGE_ROOT = os.path.dirname(__file__)
-_PROJECT_ROOT = os.path.dirname(_PACKAGE_ROOT)
-
-from lightning_utilities.core.imports import package_available  # noqa: E402
-
-_HABANA_FRAMEWORK_AVAILABLE = package_available("habana_frameworks")
-if _HABANA_FRAMEWORK_AVAILABLE:
-    from habana_frameworks.torch.utils.library_loader import is_habana_available
-
-    _HPU_AVAILABLE: bool = is_habana_available()
-else:
-    _HPU_AVAILABLE = False
-
-from lightning_habana.__about__ import *  # noqa: E402, F401, F403
-from lightning_habana.accelerator import HPUAccelerator  # noqa: E402
-from lightning_habana.plugins.io_plugin import HPUCheckpointIO  # noqa: E402
-from lightning_habana.plugins.precision import HPUPrecisionPlugin  # noqa: E402
-from lightning_habana.strategies.parallel import HPUParallelStrategy  # noqa: E402
-from lightning_habana.strategies.single import SingleHPUStrategy  # noqa: E402
-
-__all__ = [
-    "HPUAccelerator",
-    "HPUParallelStrategy",
-    "SingleHPUStrategy",
-    "HPUPrecisionPlugin",
-    "HPUCheckpointIO",
-]
+from lightning_habana.__about__ import *  # noqa: F401, F403
+from lightning_habana.utils import *  # noqa: F401, F403
```

Comment:
why import all utils?

Reply:
moving hpu_availability definitions to utils

Reply:
but is this need in root, as there is not expectation you can run this package without HPU